plugins/scaffolder-common/src/permissions.ts

```diff
@@ -30,6 +30,13 @@ export const RESOURCE_TYPE_SCAFFOLDER_TEMPLATE = 'scaffolder-template';
  */
 export const RESOURCE_TYPE_SCAFFOLDER_ACTION = 'scaffolder-action';
 
+/**
+ * Permission resource type which corresponds to a scaffolder task.
+ *
+ * @alpha
+ */
+export const RESOURCE_TYPE_SCAFFOLDER_TASK = 'scaffolder-task';
+
```

Comment:
since there are no conditions, you can skip defining a resource type in all the permissions below (which will also simplify other stuff 😄)

```suggestion
```